pygpsclient/settings_frame.py

```diff
@@ -841,6 +867,17 @@ def mapzoom(self) -> int:
 
         return self._mapzoom.get()
 
+    @property

```

Comment:
as per previous comment on scale slider. I appreciate that having the slider on the widget itself means you can't persist the scale as a standard configuration property.